modules/apps/commerce/commerce-payment-method-paypal/src/main/java/com/liferay/commerce/payment/method/paypal/internal/PayPalCommercePaymentMethod.java

```diff
@@ -838,22 +848,116 @@ public CommercePaymentResult voidTransaction(
 		return null;
 	}
 
-	private RefundRequest _buildRefundRequestBody(
-		String amount, String currencyCode) {
+	private List<PurchaseUnitRequest> _buildFullRequestBody(
+			CommerceOrder commerceOrder, Locale locale)
+		throws PortalException {
 
-		Money money = new Money();
+		List<PurchaseUnitRequest> purchaseUnits = new ArrayList<>();
```

Comment:
We should rename to this.
```suggestion
		List<PurchaseUnitRequest> purchaseUnitRequests = new ArrayList<>();
```